dataStructures/VariantProps/VariantProps.tsx

```diff
@@ -0,0 +1 @@
+export type VariantProps = { variant?: "bright" | "dark" };
```

Comment:
```suggestion
export type Variant = "bright" | "dark"
```

1. You can create a union type in this case.
2. Why you create an extra file for just one type? Is there code that will follow? Maybe you can store it in a file which is more common with some other common types. For example **/types/index.tsx**
3. If it is not for describing a component then you can write it without ...props. Now it was a bit confusing for me because i was looking for a component.

Reply:
Adding the Props was a recommendation. So I would like to use it for the moment.